### PR TITLE
New/GUM 36: add toast notifications

### PR DESCRIPTION
**Pull request by R-Mielamud**

### CODE REVIEW

File: src/helpers/notifications/notifications.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Toast from 'react-native-simple-toast';
+
+function notify(text: string) {
+  Toast.show(text, Toast.LONG);
```
Comment (yevhenprybytok) [inline src/helpers/notifications/notifications.ts]: I think we need at least a few of them (error, warning(?), info, success)

File: src/services/auth-api/index.ts [hunk 2]
```diff
@@ -20,28 +21,42 @@ class AuthApi {
...
+  ): Promise<AuthResponse | undefined> {
+    try {
+      return this.#http.load(`${this.#apiPath}${ApiPath.SIGN_IN}`, {
+        method: HttpMethod.POST,
+        contentType: ContentType.JSON,
+        payload: JSON.stringify(payload),
+        hasAuth: false,
+      });
+    } catch (err: any) {
+      notify(err?.message ?? 'Failed to log in');
```
Comment (yevhenprybytok) [inline src/services/auth-api/index.ts]: Also we need to rename it probably, to make it more clear. "Notify" could mean a lot, let's try something like 'Toaster' or similar, let's choose.

File: src/helpers/notifications/notifications.ts (hunk 1 above)
Comment (R-Mielamud) [reply src/helpers/notifications/notifications.ts]: Toast is not customizable. For example, we can't set text color or background color. So error, warning, info and success won't be different.

Comment (yevhenprybytok) [reply src/helpers/notifications/notifications.ts]: Sorry, maybe I was not very clear. We will need some notifications that can appear on top of the screen and have different styles for info / error kinds of cases

**Review by katyakostikova — APPROVED** [review]

File: ios/Podfile [hunk 3]
```diff
@@ -14,6 +14,8 @@ target 'growupmobile' do
 
   pod 'react-native-image-picker', :path => '../node_modules/react-native-image-picker'
 
+  pod 'react-native-simple-toast', :path => '../node_modules/react-native-simple-toast'
```
Comment (yevhenprybytok) [inline ios/Podfile]: It looks like this package does not allow agile customisation. Let's change this to react-native-toast-message instead.

**Review by yevhenprybytok — APPROVED** [review]

**Pull request merged**